rest/admin_api.go

```diff
@@ -533,6 +537,11 @@ func (h *handler) handlePutDbConfig() (err error) {
 			}
 
 			headerVersion := h.rq.Header.Get("If-Match")
+			headerVersion, err = h.readETag(headerVersion)
```

Comment:
Similarly, I think `readETag` should be getting the header value, rather than making every handler have to know how to use `If-Match`